WooCommerce/UITestsFoundation/Screens/Products/SingleProductScreen.swift

```diff
@@ -46,4 +46,30 @@ public final class SingleProductScreen: ScreenObject {
         XCTAssertTrue(app.staticTexts["TIP"].exists)
         XCTAssertTrue(app.textViews[productName].exists)
     }
+
+    public func verifyProductTypeScreenLoaded(productType: String) throws -> Self {
+        let priceLabel = NSPredicate(format: "label CONTAINS[c] %@", "price")
+        let inventoryLabel = NSPredicate(format: "label CONTAINS[c] %@", "inventory")
+        let productTypeLabel = NSPredicate(format: "label CONTAINS[c] %@", productType)
+        let addVariationLabel = NSPredicate(format: "label CONTAINS[c] %@", "add variations")
```

Comment:
Since this is an asserting function, I think it would benefit from more strict matching (this is a tested code):

```suggestion
        let priceLabel = NSPredicate(format: "label == 'Add Price'")
        let inventoryLabel = NSPredicate(format: "label == 'Inventory'")
        let productTypeLabel = NSPredicate(format: "label ==[c] '\(productType)'")
        let addVariationLabel = NSPredicate(format: "label == 'Add variations'")
```

Also, it would be easier to understand what exactly is missing, if an assertion fails.

Reply:
updated in https://github.com/woocommerce/woocommerce-ios/pull/8407/commits/c9a64bce98ca8c9a44005cc30b3e2e8e5a22e02b